src/containers/shared/metaParser.tsx

```diff
@@ -1,6 +1,6 @@
+import type { BaseTransaction } from 'xrpl'
 import { XRP_BASE } from './transactionUtils'
 import { ExplorerAmount } from './types'
-import { TransactionCommonFields } from './components/Transaction/types'
```

Comment:
Should this type be deleted now?

Reply:
It is still being used in other transactions as well. I will create an issue to fix all xrpl.js related types and fix them altogether